### ISSUE TITLE
Ciprofloxacin Tutorial  - question on Dose

### ISSUE THREAD

**Issue by moulirc** [Open-Systems-Pharmacology/Forum#372]
![cipro](https://user-images.githubusercontent.com/27323536/63224503-88133d80-c208-11e9-88e8-466d68953802.jpg)

I am using Ciprofloxacin data to simulate PK profile for 500 mg dose. In the Tutorial the data was extracted from Davis et al., for both IV and Oral (750 mg). Similarly, I used data from Davis et al.., for 500 mg (paper attached) to use it as a observed value. 

The observed Cmax for 750 mg is around 2.74 and the predicted is around  2.69 (Example tutorial for the website). After changing the dose to 500mg reduces the Cmax to 1.97 - while the observed is around 2.64 (Data extracted from the publication - attached). Could you please let me know why there is a big change by changing the dose amount from 750 to 500mg.

Attached is the project file and the image file. 

[PBPK_5Q.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/3513011/PBPK_5Q.zip)

[Davis_500.pdf](https://github.com/Open-Systems-Pharmacology/Forum/files/3513023/Davis_500.pdf)

**Comment by StephanSchaller** [comment]
Could be due to (inter/intra-individual) variability?
The 500mg absorption looks faster to me. This might already be enough explanation for a (relatively) higher Cmax. What are the AUC values for the two observed datasets?

**Comment by moulirc** [comment]
AUC for 750 mg is 16.47 and for 500 mg is 12.98
![PKprofile](https://user-images.githubusercontent.com/27323536/63236877-e1b84e00-c282-11e9-89bd-9c77bb35ab05.jpg)

**Comment by tobiasK2001** [comment]
Dear moulric,

you may find this paper usefull:
https://link.springer.com/content/pdf/10.1007%2Fs40262-018-0661-6.pdf

I described and updated and more universal Ciprofloxacin model compared to the version used in the WS. Further it discusses the probably a bit challenging absorption effects of the dose of Cipro and ways to overcome this.

Hope that might help you.

Best, Tobias

**Comment by JanSchlender** [comment]
Hi moulric,
The PK of Ciprofloxacin is known to be dose-linear for the therapeutic dose range. When looking at the model parametrisation, this would also be expectable with respect to the predictions. Please note, that the formulations used in the publication you refer to are different than the one used in the Davis publication for 750mg. This might explain the slightly steeper absorption for the 500mg dose.
Best,
Jan

**Comment by RobinM92** [comment]
Dear moulric,

To add to the answer of Tobias, we also recently published a PBPK model 
for cipro 
(https://www.sciencedirect.com/science/article/pii/S0928098718305244?via%3Dihub#f0030).
Although the focus here was cUTI children, adult PO data was visited 
during the model development process, and also the Davis dataset was 
used. We also saw that optimization of the dissolution parameters was 
necessary to describe the fast release from the formulation. We used a 
Weibull function with no lag time, a shape factor of 0.80 and 4 hours 
for the 50 % dissolution time, followed by transcellular intestinal 
absorption.

Good luck!
Robin

Dr. ir. Robin Michelet
Post-doctoral researcher

Freie Universitaet Berlin
Institute of Pharmacy
Dept. of Clinical Pharmacy & Biochemistry
Kelchstr. 31
12169 Berlin
Germany
Phone:  + 49 30 838 50659
Fax:      + 49 30 838 4 50656
Email: robin.michelet@fu-berlin.de
www.clinical-pharmacy.eu
